oneview_redfish_toolkit/blueprints/manager_collection.py

```diff
@@ -53,38 +47,13 @@ def get_manager_collection():
             Exception: Generic error, logs the exception and call abort(500).
     """
 
-    try:
-        # Gets all enclosures
-        enclosures = g.oneview_client.enclosures.get_all()
-
-        if not enclosures:
-            raise OneViewRedfishResourceNotFoundError(
-                "enclosures", "Resource")
-
-        # Gets all server hardware
-        server_hardware_list = g.oneview_client.server_hardware.get_all()
-
-        if not server_hardware_list:
-            raise OneViewRedfishResourceNotFoundError(
-                "server-hardware-list", "Resource")
-
-        # Build Manager Collection object and validates it
-        mc = ManagerCollection(server_hardware_list, enclosures)
+    # Gets all enclosures
+    enclosures = g.oneview_client.enclosures.get_all()
 
-        # Build redfish json
-        json_str = mc.serialize()
-        # Build response and returns
-        return Response(
-            response=json_str,
-            status=status.HTTP_200_OK,
-            mimetype="application/json")
+    # Gets all server hardware
+    server_hardware_list = g.oneview_client.server_hardware.get_all()
 
-    except OneViewRedfishResourceNotFoundError as e:
-        # In case of error print exception and abort
-        logging.exception(e)
-        abort(status.HTTP_404_NOT_FOUND, e.msg)
+    # Build Manager Collection object and validates it
+    mc = ManagerCollection(server_hardware_list, enclosures)
```

Comment:
your other PR impacts this one, a rebase will be required.

Reply:
Yes. I planned to merge #297 first 👍